MetaMask/metamask-extension
# Snow 2.0.1

> Reviewed and approved originally @ https://github.com/MetaMask/metamask-extension/pull/20471

This is an attempt to bump Snow in MetaMask from 1.5.0 to [2.0.1](https://github.com/LavaMoat/snow/pull/134).

In between Snow went through a lot. The major things to be aware of are:

* For better security, Snow will start blocking some core browser actions that aren't normally used.
* For better security, Snow expects some minimal level of CSP protection for help.
* For better security, Snow expects to be loaded and called on every HTML file served by the same origin.

### QA Help

This PR might need some QA. This introduces zero new features, only hardens security of the product.
Meaning, there isn't anything new to focus on here in regards of QA. Instead, a general QA should be taken against the app, to see that the security hardening introduced does not harm the normal already-existing functionality of the MM app.

[UPDATE: QA APPROVED](https://github.com/MetaMask/metamask-extension/pull/20471#issuecomment-1746269356)
